<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HLookSettingsForm</class>
 <widget class="QWidget" name="HLookSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>578</width>
     <height>253</height>
    </rect>
   </property>
   <property name="title">
    <string>Look Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Minimum looking time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineeditMinimumLookingTime">
          <property name="toolTip">
           <string>Subject must look at least this long for a &quot;complete look&quot; to occur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Maximum looking-away time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineeditMinimumLookingAwayTime">
          <property name="toolTip">
           <string>After looking for at least the &quot;minimum looking time&quot;, the subject must look away from the stimulus at least this long before a &quot;complete look&quot; is recorded.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="sidebar">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Habit determines whether a &amp;quot;complete look&amp;quot; has occurred using these settings. The subject must look at a stimulus for at least the &amp;quot;minimum looking time&amp;quot;. The subject can look away briefly, but if they look away for more than the &amp;quot;maximum looking-away time&amp;quot;, Habit will consider the &amp;quot;look&amp;quot; to be complete.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbxInclusiveLookTime">
        <property name="text">
         <string>Inclusive look time?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sidebar_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When the subject looks away from a stimulus for a short time (less than the &amp;quot;maximum looking-away time&amp;quot;) and then looks back to the same stimulus, should Habit count that brief time spent not looking as &amp;quot;looking time&amp;quot;? Habit 1.0 always used &amp;quot;Inclusive look time&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>52</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
